app/pb_kits/playbook/pb_card/_card.html.erb

```diff
@@ -3,7 +3,7 @@
     data: object.data,
     class: object.classname,
     aria: object.aria) do %>
-  <% if object.block %>
-    <%= pb_rails("card/card_body", props: { padding: object.padding}) { capture(&object.block) } %>
+  <% if object.children %>
+    <%= pb_rails("card/card_body", props: { padding: object.padding, children: object.children }) %>
```

Comment:
I'm not sure about calling this `children`. I understand this is what React may call it. But children represents something that is plural, and a block may represent something that is plural or singular. It makes it a little confusing. And since it is a block, and we are writing Ruby, I'd prefer to leave it named as such. We wouldn't write `block` in javascript code, because it would be confusing in that context.